instrumentation/koala/example/trace_demonstration.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'rubygems'
```

Comment:
Hmm bundle install isn't working for some reasons here

```
❯ bundle install                                                                                                                                                               ~/src/github.com/open-telemetry/opentelemetry-ruby/instrumentation/koala/example
Fetching gem metadata from https://rubygems.org/.
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Bundler could not find compatible versions for gem "opentelemetry-instrumentation-base":
  In Gemfile:
    opentelemetry-instrumentation-koala was resolved to 0.11.0, which depends on
      opentelemetry-instrumentation-base (~> 0.16.0)

Could not find gem 'opentelemetry-instrumentation-base (~> 0.16.0)', which is required by gem 'opentelemetry-instrumentation-koala', in any of the sources.
```

Reply:
tried running http_client/example, same thing.

Reply:
probably something up with my local setup

Reply:
We extracted the instrumentation base class [here](https://github.com/open-telemetry/opentelemetry-ruby/pull/698) however the gem has not been published yet so it explains why you cannot pull it.

Reply:
The gem has been published now, so this should work.